editor.planx.uk/src/@planx/components/FindProperty/model.ts

```diff
@@ -54,5 +56,8 @@ export interface SiteAddress extends MinimumSiteAddress {
 }
 
 export const DEFAULT_TITLE = "Find the property" as const;
-export const DEFAULT_NEW_ADDRESS_TITLE = "Propose a new address" as const;
+export const DEFAULT_NEW_ADDRESS_TITLE =
+  "Click or tap at where the property is on the map and name it below" as const;
+export const DEFAULT_NEW_ADDRESS_DESCRIPTION =
+  "You will need to select a location and provide a name to continue" as const;
```

Comment:
These form values also appear to be referenced in `planx-new/editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx` which is actually redundant I think.

We should be able to mark these properties as no longer optional in the `FindProperty` interface and tidy this up a bit.

Reply:
Yep good shout - cleaned up from index & Editor placeholders now !